src/paths.md

```diff
@@ -159,16 +159,17 @@ how it is resolved.
 
 ### `::`
 
-Paths starting with `::` are considered to be global paths where the segments of the path
-start being resolved from the crate root. Each identifier in the path must resolve to an
-item.
-
-> **Edition Differences**: In the 2015 Edition, the crate root contains a variety of
-> different items, including external crates, default crates such as `std` or `core`, and
-> items in the top level of the crate (including `use` imports).
+Paths starting with `::` are considered to be "global paths" where the segments of the path
+start being resolved from a place which differs based on edition. Each identifier in
+the path must resolve to an item.
+
+> **Edition Differences**: In the 2015 Edition, identifiers resolve from the "crate root"
+> (`crate::` in the 2018 edition), which contains a variety of different items, including
+> external crates, default crates such as `std` or `core`, and items in the top level of
+> the crate (including `use` imports).
 >
-> Beginning with the 2018 Edition, paths starting with `::` can only reference
-> crates in the [extern prelude].
+> Beginning with the 2018 Edition, paths starting with `::` resolve from
+> crates in the [extern prelude], i.e. they _must_ be followed by the name of a crate.
```

Comment:
```suggestion
> crates in the [extern prelude]. That is, they must be followed by the name of a crate.
```

Trying to keep italics for defining terms, not for emphasizing specific words. If you think we absolutely **must** emphasize the word, we could make it bold, but I don't think we do.